src/components/MainPage/Exporters/JsonExporter.vue

```diff
@@ -87,4 +89,10 @@ export default {
 };
 </script>
 
-<style></style>
+<style scoped>
+.theme--light.v-btn.v-btn--disabled:not(.v-btn--flat):not(.v-btn--text):not(.v-btn-outlined) {
```

Comment:
You know what?
If you want to control the style with our CSS, maybe it's time to use one of the new CSS technologies, I think we already have a `sass` packages in `package.json`.

I really don't like this CSS because it is very specific and very unclear, and later we get stuck with him.

Reply:
I haven't check yet the `sass` solution that you offer 
but I think that  also there you will need to use a selector like this because you want to catch only the disabled button

maybe the "easer" solution is to use the gray as you've offered in the previous PR